docs/source/guides/building_bentos.rst

```diff
@@ -0,0 +1,241 @@
+A Bento can be built with the ``bentoml build`` command with the ``bentofile.yaml``
+configuration file. Here's an example of that process from the `quickstart guide (https://github.com/bentoml/gallery/tree/main/quickstart#build-bento-for-deployment)_`.
+
+.. code:: yaml
+
+   service: "service:svc"
+   description: "file: ./README.md"
+   labels:
+      owner: bentoml-team
+      stage: demo
+   include:
+      - "*.py"
+   python:
+      packages:
+         - scikit-learn
+         - pandas
+
+The service field is the python module that holds the bentoml.Service
+instance.
+
+Configuring files to include
+----------------------------
+
+In the example above, the ``*.py`` is including every Python file in
+the working directory.
+
+You can also include other wildcard and directory matching.
+
+.. code:: yaml
+
+   ...
+
+   include:
+      - "data/"
+      - "**/*.py"
+      - "config/*.json"
+
+If the include field is not specified, BentoML, by default, will include
+every file in the working directory.
+
+If the user needs to include a lot of files, another approach is to
+only specify which files to be ignored.
+
+In this situation, the user can use a ``.bentoignore`` file by placing it
+in the working directory and all the files specified there will be ignored
+when building the Bento.
+
+This is what a ``.bentoignore`` file would look like.
+
+.. note::
+
+   __pycache__/
+   *.py[cod]
+   *$py.class
+   .ipynb_checkpoints/
+   training_data/
+
+Building a Bento
+================
+
+To build a Bento, simply run the following command from your project
+directory that contains your ``bentofile.yaml``:
+
+.. code:: bash
+
+   bentoml build
+
+By default, ``build`` will include all files in current working
+directory, besides the files specified in the ``.bentoignore`` file in
+the same directory. It will also automatically infer all PyPI packages
+that are required by the service code, and pin down the version used
+in current environment.
+
+Bento Format
+================
+
+BentoML is a standard file format that describes how to load and run
+a ``bentoml.Service`` defined by the user. It includes code that
+instantiates the ``bentoml.Service`` instance, as well as related
+configurations, data/model files, and dependencies.
+
+BentoML file format is a YAML file that looks like
+
+.. code:: yaml
+
+   service: "service:svc"
+   description: "file: ./README.md"
+   labels:
+      owner: bentoml-team
+      stage: demo
+   include:
+      - "*.py"
+   python:
+      packages:
+         - scikit-learn
+         - pandas
+
+Python Packages
+===============
+
+Whether you're using pip or conda, you can specify which Python packages
+to include in your Bento by configuring them in ``bentofile.yaml``.
+
+Python Options
+--------------
+
+There are two ways to specify packages in the Bentofile. First,
+we can list packages like below. When left without a version,
+pip will just use the latest release.
+
+.. code:: yaml
+
+   python:
+     packages:
```

Comment:
links can also put under package:
```yaml
packages:
  - -f ....
```

Reply:
like a link to a github repo holding a pip package?